doc/developers-guide/Hooks-and-handlers.md

```diff
@@ -106,8 +95,19 @@ Rs = mongoose_acc:get(offline_messages, Acc1, []),
 
 ```
 
+### Sidenote: something deprecated
+
+Occassionally you may find in the MongooseIM source code some calls to `ejabberd_hooks:run/3`.
```

Comment:
Occassionally you may find some calls to `ejabberd_hooks:run/3` in the MongooseIM source code.